shared/constants.js

```diff
@@ -48,8 +48,7 @@ module.exports = {
         },
         pod: {
             checkCmd: 'pod --version',
-            minVersion: '1.7.2',
-            maxVersion: '1.9.1'
```

Comment:
Where is this value used?

Reply:
forceios checks that the tool exists and has a version between min an max.
Patches are pretty frequent - so you end up with error like
`Installed pod version (1.9.3) is more than the maximum supported version (1.9.1).  Please downgrade your version of pod.`